docs/configuration.html

```diff
@@ -203,6 +203,14 @@ <h5>Updating Thread Configs</h5>
     <li><code>background.threads</code></li>
   </ul>
 
+  <h5>Updating ConnectionQuota Configs</h5>
+  The maximum number of connections allowed for a given IP/host by the broker may be updated dynamically at cluster-default level used by all brokers.
+  The changes will apply for new connection creations.
```

Comment:
Reword slightly, since the limit takes into account existing connections as well?

Reply:
@rajinisivaram  Thanks for the review. Updated the PR. Please take one more look.